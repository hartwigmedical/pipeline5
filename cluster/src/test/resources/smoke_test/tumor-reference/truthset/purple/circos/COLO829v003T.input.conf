<<include etc/colors_fonts_patterns.conf>>

show_ticks          = no
show_tick_labels    = no

<ideogram>

    <spacing>
        default = 0.002r
    </spacing>

    # Ideogram position, fill and outline
    radius           = 0.90r
    thickness        = 50p
    fill             = yes
    stroke_color     = dgrey
    stroke_thickness = 2p

    # Minimum definition for ideogram labels.

    show_label       = yes
    # see etc/fonts.conf for list of font names
    label_font       = default
    label_radius     = 1r + 30p
    label_size       = 30
    label_parallel   = yes
    label_case       = upper

</ideogram>

<colors>
    amber = 255,126,0
    cobalt= 0,71,171
</colors>

<image>
	<<include etc/image.conf>>
</image>

karyotype = data/karyotype/karyotype.human.hg19.txt

chromosomes_units           = 1000000
chromosomes_display_default = yes
chromosomes                 = -hsZ

<highlights>

	<highlight>
		file       = gaps.txt
		ideogram   = yes
		fill_color = black_a3
		z = 5
	</highlight>

</highlights>

# Default is to clip the data to proximity min/max.
<plots>

	# -----------------------------------------------------------
	# Tumor Ratio (COBALT)
	<plot>
		type = scatter
		file = COLO829v003T.ratio.circos
		r0   = 0.575r
		r1   = 0.975r
		min  = 0
		max  = 2
		color = cobalt
		glyph = circle

		<axes>
			<axis>
				color     = lblue
				thickness = 1
				spacing   = 0.25r
			</axis>
		</axes>

		<backgrounds>
			<background>
				color = vlblue_a5
			</background>
		</backgrounds>
	</plot>

	# -----------------------------------------------------------
	# Reference Ratio (Green)
	<plot>
		type = scatter
		file = COLO829v003R.ratio.circos
		r0   = 0.575r
		r1   = 0.975r
		min  = 0
		max  = 2
		color = green
		glyph = circle
	</plot>

	# -----------------------------------------------------------
	# BAF (Amber)
	<plot>
		type = scatter
		file = COLO829v003T.baf.circos
		r0   = 0.300r
		r1   = 0.550r
		min  = 0
		max  = 1
		color = amber
		glyph = circle

        <axes>
            <axis>
                color     = lorange
                thickness = 1
                spacing   = 0.25r
            </axis>
        </axes>

        <backgrounds>
            <background>
                color = vlorange_a5
            </background>
        </backgrounds>
	</plot>

</plots>

<<include etc/housekeeping.conf>>

<colors>
    chr1 = 128,125,186
    chr2 = 145,142,179
    chr3 = 161,159,173
    chr4 = 179,176,166
    chr5 = 196,193,160
    chr6 = 213,210,153

    chr7 = 230,228,147
    chr8 = 202,218,138
    chr9 = 175,209,129
    chr10 = 147,199,120
    chr11 = 120,190,111
    chr12 = 92,180,102

    chr13 = 65,171,93
    chr14 = 65,166,110
    chr15 = 65,162,128
    chr16 = 65,158,145
    chr17 = 65,154,163
    chr18 = 65,150,180

    chr19 = 66,146,198
    chr20 = 76,142,196
    chr21 = 86,139,194
    chr22 = 97,135,192
    chrX = 107,132,190
    chrY = 117,128,188
</colors>