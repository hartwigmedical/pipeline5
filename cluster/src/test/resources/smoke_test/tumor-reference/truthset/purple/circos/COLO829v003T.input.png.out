debuggroup summary 0.26s welcome to circos v0.69-6 31 July 2017 on Perl 5.024001
debuggroup summary 0.26s current working directory /
debuggroup summary 0.26s command /opt/tools/circos/0.69.6/bin/circos -nosvg -conf /data/output/circos/COLO829v003T.input.conf -outputdir /data/output/plot -outputfile COLO829v003T.input.png
debuggroup summary 0.26s loading configuration from file /data/output/circos/COLO829v003T.input.conf
debuggroup summary 0.26s found conf file /data/output/circos/COLO829v003T.input.conf
debuggroup summary 0.45s debug will appear for these features: output,summary
debuggroup summary 0.45s bitmap output image /data/output/plot/COLO829v003T.input.png
debuggroup summary 0.45s parsing karyotype and organizing ideograms
debuggroup summary 0.57s karyotype has 24 chromosomes of total size 3,095,677,436
debuggroup summary 0.58s applying global and local scaling
debuggroup summary 0.59s allocating image, colors and brushes
debuggroup summary 10.12s drawing 24 ideograms of total size 3,095,677,436
debuggroup summary 10.12s drawing highlights and ideograms
debuggroup summary 10.12s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.bg.conf
debuggroup summary 10.12s process track_0 highlight /data/output/circos/gaps.txt
debuggroup summary 10.21s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 10.21s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 10.21s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 10.22s processing track_0 scatter /data/output/circos/COLO829v003T.ratio.circos
debuggroup summary 10.23s processing track_1 scatter /data/output/circos/COLO829v003R.ratio.circos
debuggroup summary 10.25s processing track_2 scatter /data/output/circos/COLO829v003T.baf.circos
debuggroup summary 10.27s drawing track_0 scatter z 0 COLO829v003T.ratio.circos orient out
debuggroup summary 10.28s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 10.44s drawing track_1 scatter z 0 COLO829v003R.ratio.circos orient out
debuggroup summary 10.45s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 10.49s drawing track_2 scatter z 0 COLO829v003T.baf.circos orient out
debuggroup summary 10.49s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup output 10.63s generating output
debuggroup output 11.30s created PNG image /data/output/plot/COLO829v003T.input.png (448 kb)
